torchbox/torchbox.com
# PullQuote - Make Role Optional

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1405109384)

### Description of Changes Made
Small ticket which simply makes the role field on the PullQuoteBlock optional. As suggested in stand-up, I've made this PR straight to the main branch so that Lily and Lisa can start using it as soon as possible.

### How to Test
Visit any blog page, (historical) work, standard or service page and add a PullQuoteBlock to the body StreamField. You should now be able to add a quote and attribution, leaving the role field empty if desired.
### Screenshots
![image](https://github.com/torchbox/torchbox.com/assets/68611528/cbde1c1b-4419-4b3f-b93e-85f6f1b2bcbb)
![image](https://github.com/torchbox/torchbox.com/assets/68611528/47811f5b-688d-4db1-b3d3-19acfab0b6b6)

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
